doc/source/user_guide/ug_case_settings.rst

```diff
@@ -488,20 +492,20 @@ zbgc_nml
    "``mu_max_sp``", "real", "maximum growth rate small plankton per day", "0.41"
    "``nitratetype``", "real", "mobility type between stationary and mobile nitrate", "-1.0"
    "``op_dep_min``", "real", "light attenuates for optical depths exceeding min", "0.1"
-   "``phi_snow``", "real", "snow porosity for brine height tracer", "-0.3"
+   "``phi_snow``", "real", "snow porosity for brine height tracer (compute from snow density if negative)", "-1.0"
```

Comment:
What is going on here.  Our default is -0.3.  Is it the case that any negative value has the same impact?  Should we change the value to -1.0 in the code and namelist?

Reply:
Any negative value does the same thing.  Snow porosity is computed from snow density.